_data/changelogs/component-card.yml

```
@@ -2,6 +2,12 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added `usa-card__header--exdent` variant to variants table
+    affectsGuidance: true
+    githubPr: 2228
+    githubRepo: uswds-site
+    versionUswds: N.N.N
```

Added a period and removed the USWDS version.
```suggestion
  - date: NNNN-NN-NN
    summary: Added `usa-card__header--exdent` variant to variants table.
    affectsGuidance: true
    githubPr: 2228
    githubRepo: uswds-site
```